dev/PushNotifications/PushNotifications.idl

```diff
@@ -1,38 +1,63 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-namespace Microsoft.ProjectReunion
+namespace Microsoft.Windows.PushNotifications
 {
-    enum ChannelStatus
+    enum PushNotificationChannelStatus
     {
         InProgress, // The request is in progress and there is no retry operation
-        InProgressRetry, // The request is in progress and is in a backoff retry state
+        InProgressRetry, // The request is in progress and is in a backoff retry state. Check ExtendedError for HRESULT for retryable error.
```

Comment:
What do you thing of adding another state: WaitingForRetry?

Reply:
DId not get the context here? Can you summarize here please?